static/js/brand-store/Settings/Settings.tsx

```diff
@@ -19,7 +19,7 @@ import NotAuthorized from "../NotAuthorized";
 import PasswordToggle from "../shared/PasswordToggle";
 import SectionNav from "../SectionNav";
 
-import { RouteParams, Member, CurrentMember } from "../types/shared";
+import { RouteParams, Member } from "../types/shared";
```

Comment:
You can make it more clear that you're importing types by...
```suggestion
import type { RouteParams, Member } from "../types/shared";
```